src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -3006,27 +3006,23 @@ internal static Rect3D ToRect3D(this BoundingBox bounds, double minRectSize = 0.
         /// <returns>A <see cref="BoundingBox"/> object encapsulating the geometry.</returns>
         internal static BoundingBox Bounds(this GeometryModel3D geom, float defaultBoundsSize = 5.0f)
         {
-            if (geom.Geometry.Positions == null || geom.Geometry.Positions.Count == 0)
-            {
-                return new BoundingBox();
-            }
+            var bounds = geom.Bounds;
 
-            if (geom.Geometry.Positions.Count > 1)
+            if(Math.Abs(bounds.Size.LengthSquared()) < defaultBoundsSize * defaultBoundsSize * 3)
```

Comment:
A square of any number will always be greater than zero, why do we need `Math.Abs`?

Reply:
Can you explain this formula, where do you get `3` from?